src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2720,7 +2721,30 @@ public static void TestNameWithInvariantCulture()
                 Assert.True(pacific.DisplayName.IndexOf("Pacific", StringComparison.OrdinalIgnoreCase) >= 0, $"'{pacific.DisplayName}' is not the expected display name for Pacific time zone");
 
             }).Dispose();
+        }
+
+        private static bool CanTestWindowsNlsDisplayNames => s_isWindows && RemoteExecutor.IsSupported && WindowsUILanguageHelper.GetInstalledWin32CulturesWithUniqueLanguages().Length > 1;
```

Comment:
>WindowsUILanguageHelper.GetInstalledWin32CulturesWithUniqueLanguages().Length [](http://example.com/codeflow?start=106&length=77)

nit: would be better to check that inside the method to ensure calling it once.

Reply:
I could, but then the test would run every time instead of skipping when not applicable.

Reply:
does it matter much?

Reply:
I have an idea to fix that.  Thanks.